jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -96,15 +98,69 @@ pipeline {
             '''
             }
         }
-        stage ('Destroy AWS infrastructure') {
+        stage ('Change root password') {
```

Comment:
can you change stage name to `EC2 Connection Prerequisites`

Reply:
Done